### DISCUSSION TITLE
sudo apt install fails with Failed to fetch http://security.ubuntu.com/... 404  Not Found [IP: ...

### DISCUSSION THREAD

**Discussion by skirpichev** [community/community#26649] [Actions]
<p>See e.g. <a href="https://github.com/diofant/diofant/commit/1e643827e47b4f91fda9a7954fe99dee235693bd/checks?check_suite_id=334267422" rel="nofollow noopener">the latest build</a>:</p>
<pre><code>sudo apt install graphviz texlive-xetex texlive-fonts-recommended
                        fonts-freefont-otf latexmk lmodern
...
Err:34 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 ruby2.5 amd64 2.5.1-1ubuntu1.5
404 Not Found [IP: 52.168.50.79 80]
http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libruby2.5 amd64 2.5.1-1ubuntu1.5
404 Not Found [IP: 52.168.50.79 80]
...
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/r/ruby2.5/ruby2.5_2.5.1-1ubuntu1.5_amd64.deb 404 Not Found [IP: 52.168.50.79 80]
Fetched 140 MB in 5s (25.9 MB/s)
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/r/ruby2.5/libruby2.5_2.5.1-1ubuntu1.5_amd64.deb 404 Not Found [IP: 52.168.50.79 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
</code></pre>
<p>My workflow: <a href="https://github.com/diofant/diofant/blob/master/.github/workflows/test.yml" rel="nofollow noopener">test.yml</a></p>

**Comment by wipe2238** [comment, marked as answer]
<p>Try <code>sudo apt update</code> before installing.</p>

**Reply by KiarieWinfred** [reply]
awesome. Thanks

**Reply by Alethiea-Cheng** [reply]
thank you hello kitty

**Comment by aliathegreat** [comment]
<p>my problem solved. thx u so much</p>

**Comment by sheikhuzair26** [comment]
<p>Thanks Alot &lt;3</p>

**Comment by satyamagrawalread** [comment]
<p>Now, its’ working<br>
Thanks</p>

**Comment by steveipkis** [comment]
<p>Resolved this in docker by running:</p>
<p><code>docker system prune</code></p>

**Comment by ettaboyle** [comment]
<p>Nice!! Thanks for adding the additional option <a class="mention" href="/steveipkis">@steveipkis</a> + welcome to the community forum :slight_smile:</p>
